### PR TITLE
M sa2 15 20

### PR DESCRIPTION
**Pull request by CHESTER-ART**
need to check please

### CODE REVIEW

File: Ryabuhin Artiom/02.git.local/02.GIT.Local.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+	bash
```
Comment (pluhin) [inline Ryabuhin Artiom/02.git.local/02.GIT.Local.md:1]: Wrong markdown, 

Please check documentation for code sections
https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks

and https://guides.github.com/features/mastering-markdown/#examples

File: Ryabuhin Artiom/02.git.local/git_tree.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+*   c7fe392 (HEAD -> master) something went wrong ((
```
Comment (pluhin) [inline Ryabuhin Artiom/02.git.local/git_tree.txt:1]: Please provide separate output for each branch

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Ryabuhin Artiom/02.git.local/git_tree.txt (hunk 2 above)
Comment (CHESTER-ART) [reply Ryabuhin Artiom/02.git.local/git_tree.txt]: please tell me the command for output only for each branch.
I know how to show all commits only.

File: Ryabuhin Artiom/02.git.local/02.GIT.Local.md (hunk 1 above)
Comment (CHESTER-ART) [reply Ryabuhin Artiom/02.git.local/02.GIT.Local.md]: Done

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**